packages/legend-application-studio/style/components/editor/_service-editor.scss

```diff
@@ -103,6 +103,18 @@
     background: var(--color-yellow-200);
   }
 
+  &__tab__hint {
+    @include flexCenter;
+
+    border-radius: 0.2rem;
+    margin-left: 1rem;
```

Comment:
very minor, I think 0.5rem might be better for spacing than 1rem, like how info icon for text-search-advanced-config__panel__header__hint has 0.5 padding. But that is subjective I think ^^